### ISSUE TITLE
Would be nice to have reusable log formats

### ISSUE THREAD

**Issue by zizzpudding** [grafana/explore-logs#917]
Many users have asked for reusable log formats.  Especially helpful for complex JSON line formats. 

It would be even nicer to have log formats that could be shared across Grafana between different applications (ex.  Classic Explore, Explore Apps and App O11y)

**Comment by gtk-grafana** [comment]
Addressed by https://github.com/grafana/logs-drilldown/pull/1664. @matyax do you think this can be closed by that issue?

**Comment by matyax** [comment]
Yes, I think so, specially if https://github.com/grafana/logs-drilldown/pull/1664 supports parsed fields.
